src/main/java/com/juniorstart/juniorstart/model/Technologies.java

```diff
@@ -0,0 +1,30 @@
+package com.juniorstart.juniorstart.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Setter
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Technologies {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @NotNull
+    private String nameTechnology;
+
+    @JsonIgnore
+    @ManyToMany(mappedBy = "technologies")
+    private Set<JobOffer> jobOffers = new HashSet<>();
```

Comment:
Tu chyba to samo. Według mnie wystarczy, że oferta posiada jakieś technologie i tyle. Technologia nie musi mieć przypisania do ofert według mnie.

Reply:
też się zgadzam, że niepotrzebne jest to.

Reply:
+ metody addJobOffer itp, bo @getter i @setter generuje je.